Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -0,0 +1,2138 @@
+            mal = {
+                'Malicious': {
+                    'Vendor': 'ThreatConnect',
+                    'Description': desc,
+                }
+            }
+            if indicator_type == 'ip':
+                mal['Address'] = value
+
+            elif indicator_type == 'file':
+                mal['MD5'] = value
+                mal['SHA1'] = ind.get('sha1')
+                mal['SHA256'] = ind.get('sha256')
+
+            elif indicator_type == 'url':
+                mal['Data'] = value
+
+            elif indicator_type == 'domain':
+                mal['Name'] = value
+
+            context_path = outputPaths.get(indicator_type)
+            if context_path is not None:
+                context[context_path].append(mal)
+
+        elif int(rating) >= 1:
+            dbot_score = 2
+        else:
+            dbot_score = 1
+
+        if include_dbot_score:
+            context['DBotScore'].append({
+                'Indicator': value,
+                'Score': dbot_score,
+                'Type': indicator_type,
+                'Vendor': 'ThreatConnect'
+            })
+
+        context['TC.Indicator(val.ID && val.ID === obj.ID)'].append({
+            'ID': ind['id'],
+            'Name': value,
+            'Type': ind['type'],
+            'Owner': ind['ownerName'],
+            'Description': ind.get('description'),
+            'CreateDate': ind['dateAdded'],
+            'LastModified': ind['lastModified'],
+            'Rating': rating,
+            'Confidence': confidence,
+
+            # relevant for domain
+            'Active': ind.get('whoisActive'),
+
+            # relevant for file
+            'File.MD5': ind.get('md5'),
+            'File.SHA1': ind.get('sha1'),
+            'File.SHA256': ind.get('sha256'),
+        })
+
+        if 'group_associations' in ind:
+            if len(ind['group_associations']) >= 0:
+                context['TC.Indicator(val.ID && val.ID === obj.ID)'][0]['IndicatorGroups'] = ind['group_associations']
+
+        if 'indicator_associations' in ind:
+            if len(ind['indicator_associations']) >= 0:
+                context['TC.Indicator(val.ID && val.ID === obj.ID)'][0]['IndicatorAssociations'] = ind[
+                    'indicator_associations']
+
+        if 'indicator_tags' in ind:
+            if len(ind['indicator_tags']) >= 0:
+                context['TC.Indicator(val.ID && val.ID === obj.ID)'][0]['IndicatorTags'] = ind['indicator_tags']
+
+        if 'indicator_observations' in ind:
+            if len(ind['indicator_observations']) >= 0:
+                context['TC.Indicator(val.ID && val.ID === obj.ID)'][0]['IndicatorsObservations'] = ind[
+                    'indicator_observations']
+
+    context = {k: createContext(v, removeNull=True)[:MAX_CONTEXT] for k, v in context.items() if len(v) > 0}
+    return context, context.get('TC.Indicator(val.ID && val.ID === obj.ID)', [])
+
+
+def get_xindapi(indicator_value, indicatorType, owner, tc):
```

Comment:
1) add doc-string, what is `xindapi`?
2) use snake_case for variables instead of camelCase (`indicatorType` -> `indicator_type`)
3) as coding-convention, self/object instance should be the first argument. in this case, `tc` is the object instance.